docs/recipes/intregration-with-rxangular.md

```diff
@@ -0,0 +1,76 @@
+# Integration with RxAngular
+There are use cases when is desired to have a transient component state has it's lifetime bound to the component lifecycle.
+[RxAngular](https://github.com/rx-angular/rx-angular) provides a solution for this problem that is fully reactive and with focus on runtime performance and template rendering.
+
+In order to leverage this library and all the power of a global state management is possible to integrate RxAngular with NGXS.
+First let's bind a value provinient from NGXS state to RxAngular component state: 
```

Comment:
"provinient"?
Not sure what this means?

Reply:
it was a typo actually 
```suggestion
First, let's bind a value from NGXS state to RxAngular component state: 
```